src/test/regress/expected/multi_colocation_utils.out

```diff
@@ -535,17 +602,51 @@ SELECT create_reference_table('table2_groupF');
 (1 row)
 
 -- check metadata
-SELECT * FROM pg_dist_colocation 
-    WHERE colocationid >= 1 AND colocationid < 1000 
+SELECT * FROM pg_dist_colocation
+    WHERE colocationid >= 1 AND colocationid < 1000
     ORDER BY colocationid;
- colocationid | shardcount | replicationfactor | distributioncolumntype 
```

Comment:
I think we have two options here (examples below). I slightly prefer the first one because it makes easier to compare colocation groups, and you can compare table separately. What do you think?

i. Running two separate queries;

SELECT * FROM pg_dist_colocation
    WHERE colocationid >= 1 AND colocationid < 1000
    ORDER BY colocationid;
 colocationid | shardcount | replicationfactor | distributioncolumntype | defaultgroup 
--------------+------------+-------------------+------------------------+--------------
            2 |          2 |                 1 |                     23 | t
            3 |          2 |                 2 |                     25 | t
            4 |          4 |                 2 |                     23 | t
            5 |          2 |                 2 |                     23 | f
            6 |          2 |                 2 |                     23 | f
            7 |          2 |                 2 |                     23 | t
            8 |          3 |                 2 |                     23 | t
            9 |          3 |                 2 |                     23 | f
           10 |          1 |                 2 |                     23 | t
(9 rows)

SELECT logicalrelid, colocationid FROM pg_dist_partition
    WHERE colocationid >= 1 AND colocationid < 1000
    ORDER BY colocationid;
           logicalrelid           | colocationid 
----------------------------------+--------------
 table1_groupb                    |            2
 table2_groupb                    |            2
 table2_groupc                    |            3
 table1_groupc                    |            3
 table1_groupd                    |            4
 table2_groupd                    |            4
 table3_groupd                    |            4
 table3_groupe                    |            5
 table1_groupe                    |            5
 table2_groupe                    |            5
 schema_collocation.table4_groupe |            5
 table4_groupe                    |            5
 table2_group_none_1              |            6
 table1_group_none_1              |            6
 table1_group_none_2              |            7
 table1_group_none_3              |            8
 table1_group_default             |            9
 table1_groupf                    |           10
 table2_groupf                    |           10
(19 rows)

ii. or running a single query
SELECT logicalrelid, pg_dist_colocation.colocationid, shardcount, replicationfactor, distributioncolumntype, defaultgroup
    FROM pg_dist_colocation, pg_dist_partition
    WHERE pg_dist_colocation.colocationid >= 1 AND pg_dist_colocation.colocationid < 1000 AND pg_dist_colocation.colocationid = pg_dist_partition.colocationid
    ORDER BY pg_dist_colocation.colocationid, logicalrelid;
           logicalrelid           | colocationid | shardcount | replicationfactor | distributioncolumntype | defaultgroup 
----------------------------------+--------------+------------+-------------------+------------------------+--------------
 table1_groupb                    |            2 |          2 |                 1 |                     23 | t
 table2_groupb                    |            2 |          2 |                 1 |                     23 | t
 table1_groupc                    |            3 |          2 |                 2 |                     25 | t
 table2_groupc                    |            3 |          2 |                 2 |                     25 | t
 table1_groupd                    |            4 |          4 |                 2 |                     23 | t
 table2_groupd                    |            4 |          4 |                 2 |                     23 | t
 table3_groupd                    |            4 |          4 |                 2 |                     23 | t
 table1_groupe                    |            5 |          2 |                 2 |                     23 | f
 table2_groupe                    |            5 |          2 |                 2 |                     23 | f
 table3_groupe                    |            5 |          2 |                 2 |                     23 | f
 schema_collocation.table4_groupe |            5 |          2 |                 2 |                     23 | f
 table4_groupe                    |            5 |          2 |                 2 |                     23 | f
 table1_group_none_1              |            6 |          2 |                 2 |                     23 | f
 table2_group_none_1              |            6 |          2 |                 2 |                     23 | f
 table1_group_none_2              |            7 |          2 |                 2 |                     23 | t
 table1_group_none_3              |            8 |          3 |                 2 |                     23 | t
 table1_group_default             |            9 |          3 |                 2 |                     23 | f
 table1_groupf                    |           10 |          1 |                 2 |                     23 | t
 table2_groupf                    |           10 |          1 |                 2 |                     23 | t
(19 rows)